tests/test_plotting.py

```diff
@@ -398,45 +459,48 @@ def test_plotly_future_reg():
     future = m.make_future_dataframe(df=df, regressors_df=regressors_df_future, n_historic_predictions=10, periods=50)
     forecast = m.predict(df=future)
 
-    fig1 = m.plot(forecast, plotting_backend="plotly")
-    fig2 = m.plot_components(forecast, plotting_backend="plotly")
-    fig3 = m.plot_parameters(plotting_backend="plotly")
+    fig1 = m.plot(forecast, plotting_backend=plotting_backend)
+    fig2 = m.plot_components(forecast, plotting_backend=plotting_backend)
+    fig3 = m.plot_parameters(plotting_backend=plotting_backend)
 
     if PLOT:
         fig1.show()
         fig2.show()
         fig3.show()
 
 
-def test_plotly_uncertainty():
-    log.info("testing: Plotting with plotly")
+@pytest.mark.parametrize(*decorator_input)
+def test_plot_uncertainty(plotting_backend):
+    log.info(f"testing: Plotting with uncertainty without forecast in focus with {plotting_backend}")
```

Comment:
Change to:

`log.info(f"testing: Plotting with uncertainty estimation without forecast in focus with {plotting_backend}")`

Also, what do you mean by `without forecast in focus`?

Reply:
@Kevin-Chen0 I adjust the log.info messages as follows: 
`log.info(f"testing: Plotting with uncertainty estimation with {plotting_backend}")`
and
`log.info(f"testing: Plotting with uncertainty estimation with highlighted forecast step with {plotting_backend}")`
What do you think?